pkg/controller/networkpolicy/clusternetworkpolicy_test.go

```diff
@@ -446,6 +454,281 @@ func TestAddCNP(t *testing.T) {
 		expAppliedToGroups int
 		expAddressGroups   int
 	}{
+		{
+			name: "application-tier-policy",
+			inputPolicy: &secv1alpha1.ClusterNetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{Name: "cnpA", UID: "uidA"},
+				Spec: secv1alpha1.ClusterNetworkPolicySpec{
+					AppliedTo: []secv1alpha1.NetworkPolicyPeer{
+						{PodSelector: &selectorA},
+					},
+					Priority: p10,
+					Tier:     "Application",
```

Comment:
nit: maybe test with tier not specified here. Getting tier from name workflow would be covered in other cases.

Reply:
that workflow is being tested by "rules-with-same-selectors" and other test cases within for free .. where the tier is not specified but the expected policies check for the application tier to be set.

Reply:
Got it, thanks.